app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateEmojiIdFragment.kt

```diff
@@ -131,12 +141,96 @@ class CreateEmojiIdFragment : BaseFragment() {
-        startActivity(intent)
-        activity?.finish()
+    @OnClick(R.id.create_emoji_id_btn)
+    fun onCreateEmojiIdButtonClick() {
+        showEmojiWheelAnimation()
+    }
+
+    private fun showEmojiWheelAnimation() {
+        emojiWheelAnimView.playAnimation()
+
+        val fadeOutAnimnAnim = ValueAnimator.ofFloat(1f, 0f)
+        fadeOutAnimnAnim.addUpdateListener { valueAnimator: ValueAnimator ->
+            val alpha = valueAnimator.animatedValue as Float
+            nerdFaceEmoji.alpha = alpha
+            awesomeText.alpha = alpha
+            createYourEmojiIdText.alpha = alpha
+            walletAddressDescText.alpha = alpha
+            createEmojiIdButton.alpha = alpha
+        }
+        fadeOutAnimnAnim.addListener(object : AnimatorListenerAdapter() {
+            override fun onAnimationEnd(animation: Animator?) {
+                super.onAnimationEnd(animation)
+                createEmojiIdButton.visibility = View.GONE
+                awesomeTextBackView.visibility = View.GONE
+                justSecBackView.visibility = View.GONE
+                createEmojiIdTextBackView.visibility = View.GONE
+            }
+        })
+
+        fadeOutAnimnAnim.startDelay = CreateEmojiId.walletCreationFadeOutAnimDelayMs
+        fadeOutAnimnAnim.duration = CreateEmojiId.walletCreationFadeOutAnimDurationMs
+        fadeOutAnimnAnim.start()
+
+        uiHandler.postDelayed(
+            { startYourEmojiIDViewAnimation() },
+            emojiWheelAnimView.duration - CreateEmojiId.awesomeTextAnimDurationMs
+        )
+    }
+
+    private fun startYourEmojiIDViewAnimation() {
+        walletAddressDescText.text =
+            getString(R.string.create_emoji_id_set_of_emoji_your_wallet_address_desc)
+        val buttonOffset = -(continueButton.height + createEmojiButtonBottomMargin).toFloat()
+        val buttonTransAnim: ObjectAnimator =
+            ObjectAnimator.ofFloat(continueButton, View.TRANSLATION_Y, 0f, buttonOffset)
+
+        val buttonFadeInAnim = ValueAnimator.ofFloat(0f, 1f)
+        buttonFadeInAnim.addUpdateListener { valueAnimator: ValueAnimator ->
+            val alpha = valueAnimator.animatedValue as Float
+            continueButton.alpha = alpha
+        }
+
+        val buttonAnimSet = AnimatorSet()
+        buttonAnimSet.playTogether(buttonTransAnim, buttonFadeInAnim)
+        buttonAnimSet.duration = CreateEmojiId.continueButtonAnimDurationMs
+
+        val emojiTransAnim = ValueAnimator.ofFloat(2f, 1.5f)
+        emojiTransAnim.addUpdateListener { animation ->
+            val scale = animation.animatedValue.toString().toFloat()
+            emojiContainerImage.scaleX = scale
+            emojiContainerImage.scaleY = scale
+        }
+        emojiTransAnim.startDelay = CreateEmojiId.emojiIdImageViewAnimDelayMs
+
+        val titleOffset = -(yourEmojiTitleText.height).toFloat()
+        val yourEmojiIdTitleAnim =
+            ObjectAnimator.ofFloat(yourEmojiTitleText, View.TRANSLATION_Y, 0f, titleOffset)
```

Comment:
Anim's name is yourEmojiId... and text view's name is yourEmojiTitle, I think these should be the same otherwise it looks like they're not related.